apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -30,13 +88,40 @@ const linkStoryMeta: LinkStoryMeta = {
 
 export default linkStoryMeta;
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({ }: LinkProps): TemplateResult => html`
-  <pie-link></pie-link>
-`;
+const Template = ({
+    href, target, rel, size, variant, isBold, isStandalone, slot,
+}: LinkProps): TemplateResult => html`
+        <pie-link
+            variant="${variant}"
+            size="${size}"
+            href="${ifDefined(href || undefined)}"
```

Comment:
Can these be `... || nothing`?

Reply:
I'd look into the `?` and `.` attribute modifiers in Lit templating

Reply:
`... ||` nothing works like a charm, thank youuu